fastlane/Fastfile

```diff
@@ -179,6 +178,40 @@ platform :android do
     push_git_tags(tag: version_number)
   end
 
+  desc "Build purchase tester app bundle"
+  lane :build_purchase_tester_bundle do |options|
+    build_number = ENV['CIRCLE_BUILD_NUM'] || UI.user_error!('Could not find CircleCI build number.')
+    version_name = current_version_number
```

Comment:
I decided to use the version of the SDK as the version number of the purchase tester app. That way it would be easier to identify what version it's using.